adapters/Vungle/VungleAdapter/VungleRouterConfiguration.m

```diff
@@ -0,0 +1,50 @@
+//
+//  VungleRouterConfiguration.m
+//  VungleAdapter
+//
+//  Created by Akifumi Shinagawa on 2/12/19.
+//  Copyright © 2019 Vungle. All rights reserved.
+//
+
+#import "VungleRouterConfiguration.h"
+#import <VungleSDK/VungleSDK.h>
+
+// These keys are also defined in VNGPersisteceManager.
+static NSString *const kAdapterMinimumFileSystemSizeForInit = @"vungleMinimumFileSystemSizeForInit";
+static NSString *const kAdapterMinimumFileSystemSizeForAdRequest =
+    @"vungleMinimumFileSystemSizeForAdRequest";
+static NSString *const kAdapterMinimumFileSystemSizeForAssetDownload =
+    @"vungleMinimumFileSystemSizeForAssetDownload";
+
+@implementation VungleRouterConfiguration
+
++ (void)setPublishIDFV:(BOOL)publish {
+  [VungleSDK setPublishIDFV:publish];
+}
+
++ (void)setMinSpaceForInit:(int)size {
+  if (size && size >= 0) {
```

Comment:
Use `size >= 0` ?
Same for the ones below.

Reply:
@joshuajplagonera Thanks, we will update the code to use one. :)

Reply:
@joshuajplagonera Thanks, done. 😊